shims/spark301db/src/main/scala/com/nvidia/spark/rapids/shims/spark301db/Spark301dbShims.scala

```diff
@@ -105,7 +105,7 @@ class Spark301dbShims extends Spark301Shims {
         "Databricks-specific window function exec, for \"running\" windows, " +
             "i.e. (UNBOUNDED PRECEDING TO CURRENT ROW)",
         ExecChecks(
-          TypeSig.commonCudfTypes + TypeSig.DECIMAL +
+          TypeSig.commonCudfTypes + TypeSig.DECIMAL + TypeSig.Null +
```

Comment:
found a typo here. I will draft a fix

Reply:
Oh, yikes! 
I noticed that when testing on Databricks 7.3. I honestly thought I'd fixed that already.

I can post a fix shortly.

Reply:
https://github.com/NVIDIA/spark-rapids/pull/2729 just post one, can you help take a look? thx~